definitions/npm/react-select_v1.x.x/flow_v0.104.x-/react-select_v1.x.x.js

```diff
@@ -170,7 +170,7 @@ declare module 'react-select' {
     wrapperStyle?: {...},
     isSearchable?: boolean,
     // whether to cache the search results
-    cache?: boolean
+    cache?: {...} | false
```

Comment:
This is an invalid syntax and causing issue when run linter.

```
ESLint: 
Parsing error: Unexpected token

  176 |     isSearchable?: boolean,
  177 |     // whether to cache the search results
> 178 |     cache?: {...} | false
      |                 ^
  179 |   |};
  180 | 
  181 |   declare type AsyncProps = {|
```

I think the value should be `{} | false`

Reply:
Hello, two things here:

1. You shouldn't lint your `flow-typed` folder. 
2. It's a valid syntax, just make sure you have the proper ESLint and sub dependencies versions.